src/components/Activities/Activity1/Game2/components/GameSlide.tsx

```diff
@@ -20,6 +20,8 @@ interface GameSlideProps {
 
 function GameSlide(props: GameSlideProps): JSX.Element {
   const [incorrect, setIncorrect] = useState(false);
+  const [alienState, setAlienState] = useState(ALIEN_STATE.BASE);
+  const alienTimeout = useRef<Timeout>(undefined);
```

Comment:
nit: maybe my vscode squigglies are wrong, but my vscode gets a little angy abt this, maybe this could work?
```suggestion
  const alienTimeout = useRef<NodeJS.Timeout|undefined>(undefined);
```

Reply:
O I actually tried the NodeJS.Timeout thing.. but then my vscode bugged to it.. but I can double check

Reply:
hhmmmmmmmm oh wells its ok my computer just mad apparently

Reply:
or we can do this https://stackoverflow.com/questions/54934801/what-type-should-timeout-be-defined-at-in-typescript
 timeout: ReturnType<typeof setTimeout>;